test/query.test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-env jest */
+const { promisify } = require('util')
+const exec = promisify(require('child_process').exec)
+const { PostgresRepository } = require('../src/postgresRepository')
+
+const CONTAINER_NAME = 'node-dry-pg-test-database'
+const DATABASE_PORT = 5432
+const DATABASE_USER = 'node-dry-pg-test'
+const DATABASE_PASSWORD = 'test-pwd'
+const TIMEOUT = 30000
+
+const POOL_CONFIG = {
+  host: 'localhost',
+  port: DATABASE_PORT,
+  user: DATABASE_USER,
+  password: DATABASE_PASSWORD,
+  database: DATABASE_USER
+}
+
+async function startDatabase () {
+  await exec(`docker run -p ${DATABASE_PORT}:5432 --name ${CONTAINER_NAME} -d ` +
+    `-e POSTGRES_USER=${DATABASE_USER} -e POSTGRES_PASSWORD=${DATABASE_PASSWORD} ` +
+    'postgres:13-alpine')
+}
+
+describe('node-dry-pg query test', () => {
+  let postgresRepository
+
+  beforeEach(() => {
+    postgresRepository = new PostgresRepository(POOL_CONFIG)
+  })
+
+  afterEach(async () => {
+    await postgresRepository.close()
+    await exec(`docker stop ${CONTAINER_NAME}`)
+    await exec(`docker rm ${CONTAINER_NAME}`)
+  })
+
+  test('simple query', async () => {
+    await startDatabase()
+    await new Promise(resolve => setTimeout(resolve, 5000))
```

Comment:
We should use some variable here instead of magic number.
Should we also use the sleep function from the basics package?

Reply:
Yes, using the `sleep` function is a nice idea. But I would not use variables for these magic numbers as they are highly dependent on the individual test case.

Reply:
Well, then at least name it somehow as variable so that we know what the timeout intends to do :) like `databseStartupTime`

Reply:
I have put all the magic numbers in variables.